rfcs/0000-upgradeable-lifecycle.md

```diff
@@ -9,11 +9,11 @@ A simple, monolithic approach that would keep a single lifecycle version for the
 
 1. Create a command in `kp` that would give users the ability to manually update their lifecycle:
 
-	`kp lifecycle upgrade --image "gcr.io/my-registry/lifecycle:v0.9.2"`
+	`kp lifecycle upgrade --linux "gcr.io/my-registry/lifecycle:v0.9.2" --windows "gcr.io/my-registry/windows-lifecycle:v0.9.2"`
```

Comment:
The active windows pr uses a single image for the lifecycle: https://github.com/pivotal/kpack/pull/546.